src/styles/calendar.scss

```diff
@@ -0,0 +1,34 @@
+@import "./mixins";
+@import "./variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: $calendar-width;
+
+  &_cell {
+    @include square($cell-size);
+    background-color: $cell-background-color;
+
+    box-sizing: border-box;
+    border: $cell-border;
+    transition: all 0.5s linear;
+    
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    &::before {
+      display: flex;
+
+      color: black;
+    }
+    &:hover {
+      background-color: pink;
+      transform: translate(0, -20px);
+      transition: all 0.5s linear;
+      cursor: pointer;
+    }
+  }
```

Comment:
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/223091204-9d898d37-f4ff-445f-9099-a9b5b2aafb33.png)

also, don't forget to create variables for all colors

```suggestion
    align-items: center;

    &::before {
      display: flex;

      color: black;
    }
    
    &:hover {
      background-color: $c-pink;
      transform: translate(0, -20px);
      cursor: pointer;
    }
  }
```